app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,29 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>BudgetApp</title>
+    <title>Budget App</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag "forms", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag "main", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+    <script src="https://kit.fontawesome.com/f0e9415ee2.js" crossorigin="anonymous"></script>
   </head>
 
   <body>
-    <%= yield %>
+    <% if user_signed_in? %>
+      <header>
+        <p><b><%= current_user.name %></b></p>
+        <div>
+          <%= button_to 'Log out', destroy_user_session_path, method: :delete %>
+        </div>
```

Comment:
- [ ] When I try to sign out. I'm getting errors and I am unable to log out from the application. Please fix that.

![Screenshot from 2023-03-12 00-33-33](https://user-images.githubusercontent.com/26767383/224507235-60974b6c-4a53-4f45-921d-132a2d77da4a.png)

Reply:
please is there any way you could help me on this I actually suffered to get to this point please @Deepakdanger